### DISCUSSION TITLE
Idea: Athletics view

### DISCUSSION THREAD

**Discussion by hawkrives** [StoDevX/AAO-React-Native#5542] [Ideas]
I know, I know, Athletics… but there are people who would like to be better able to see what's going on in the athletics department, even if we ourselves never go to any games. 

Especially parents. Parents really like the Athletics things. 

Drew once had a branch that started this work; we'll have to see where he left off.

**Comment by drewvolz** [comment]
I had the games and their statuses. It would be neat to parse things like current team rosters, and allow you to drill down into a sport’s games, history, roster, etc.

**Comment by drewvolz** [comment]
Partial implementation lives in the branch `athletics` and can be seen at #1103

**Comment by drewvolz** [comment]
Full implementation merged into Carls at https://github.com/carls-app/carls/tree/expo/app/views/athletics

**Comment by drewvolz** [comment]
https://github.com/StoDevX/AAO-React-Native/pull/7573
